### PR TITLE
Update docs for JuliaPlots/Plots.jl/pull/3591

### PR DESCRIPTION
**Pull request by t-bltg**
Add an example of how to use `extra_kwargs` using GR.

### CODE REVIEW

**Comment by BeastyBlacksmith** [comment]
Since the gr backend doesn't have a general extra_kwargs mechanism it makes sense to list all the implemented keywords with their function

**Comment by t-bltg** [comment]
> Since the gr backend doesn't have a general extra_kwargs mechanism it makes sense to list all the implemented keywords with their function

In what format ?

Do you mean at the end in  https://docs.juliaplots.org/latest/generated/gr/ ?

**Comment by BeastyBlacksmith** [comment]
> Do you mean at the end in https://docs.juliaplots.org/latest/generated/gr/ ?

No, I'd suggest a markdown table in this section

**Comment by t-bltg** [comment]
Something like this ?

File: docs/src/backends.md [hunk 1]
```diff
@@ -90,6 +90,30 @@ Cons:
...
+  nx=50, ny=50, display_option=GR.OPTION_SHADED_MESH,
+)
+```
+
+Supported GR `extra_kwargs` :
+
+| Keyword | Description |
+| ------- | ----------- |
+| nx | Number of interpolation points in the x direction |
+| ny | Number of interpolation points in the y direction |
```
Comment (BeastyBlacksmith) [inline docs/src/backends.md:113]: Thats good, the only thing that is missing is to which seriestypes this has an effect

**Pull request merged**